### ISSUE TITLE
Component Docs Drive

### ISSUE THREAD

**Issue by rachelnabors** [facebook/react-native-website#1579]
**🍾🎉🎈Thank you everyone! We are mission complete!🍾🎉🎈**

We have a lot of component and API docs that need updating! The good news is that updating these docs can be a fun way to become more familiar with React Native’s core code while contributing to docs that help thousands of learners around the world! If this looks intimidating, we have you covered! There’s a handy guide below that should help you out :)

## What needs updating?

* Update the API to reflect the props and methods reflected in core
* Ensure example code is 1) in a Snack player and 2) uses function components and Hooks:
    * Add example code if it is missing
    * Update example code to use function components and React Hooks
    * Put example code in an embedded Snack player

## How to contribute?

1. First, select a component or API from the table below. 
2. Choose a single update to make:
    1. Update the API
    2. Add missing example code
    3. Convert existing example code to use function components and React Hooks
    4. Embed any existing example code in a Snack player
3. Comment below on what component/API you want to take on and which fix; I'll update the matrix with your name
4. Submit your PR tagged with this issue

When your PR is merged, we’ll convert your name into a checkmark, indicating completion!

### Update the API to reflect the props and methods reflected in core

This part requires a little investigating! Check [react-native/Libraries/](https://github.com/facebook/react-native/tree/9caf99162dcd1ee73c571c494ca3115540f45f69/Libraries)YourComponent and** **[react-native/Libraries/Components/](https://github.com/facebook/react-native/tree/9caf99162dcd1ee73c571c494ca3115540f45f69/Libraries/Components)YourComponent for your component or API’s list of props. Sometimes they can be hidden, like the [LayoutProps API](https://facebook.github.io/react-native/docs/layout-props) is in [react-native/Libraries/StyleSheet/StyleSheetTypes.js](https://github.com/facebook/react-native/blob/8553e1acc4195479190971cc7a3ffaa0ed37a5e0/Libraries/StyleSheet/StyleSheetTypes.js). 

> **Pro tip:** If you get stuck, search for “YourComponent.js” or search the “YourComponent” with a prop or method already listed.

### Add missing example code

If there is no example code, add a new example using [function components](https://reactjs.org/docs/hooks-state.html#hooks-and-function-components) and [Hooks](https://reactjs.org/docs/hooks-intro.html). Embed your code with a Snack player like so:

```` 
```SnackPlayer name=Component/API%20Example

// Example code goes here

``` 
````

### Convert existing example code to use function components and Hooks

If there’s a class component example already, use this [handy guide on converting class components to function components and Hooks](https://scotch.io/bar-talk/convert-class-components-to-functional-components-in-a-react-project-solution-to-code-challenge-14) to make a function component example. Then use the following code snippet to keep both function and class component examples (please remove the first and final ` ``` `—markdown formating trouble!): 

````
```
<div class="toggler">
  <ul role="tablist" class="toggle-syntax">
    <li id="functional" class="button-functional" aria-selected="false" role="tab" tabindex="0" aria-controls="functionaltab" onclick="displayTabs('syntax', 'functional')">
      Function Component Example
    </li>
    <li id="classical" class="button-classical" aria-selected="false" role="tab" tabindex="0" aria-controls="classicaltab" onclick="displayTabs('syntax', 'classical')">
      Class Component Example
    </li>
  </ul>
</div>

<block class="functional syntax" />

```SnackPlayer name=Component/API%20Function%20Component%20Example

// Function Component Example goes here

```

<block class="classical syntax" />

```SnackPlayer name=Component/API%20Class%20Component%20Example

// Class Example goes here

```

<block class="endBlock syntax" />
```
````

If there isn’t a class component example or there was already a function component example, just use the following code to embed just a function component example.

### **Embed any example code with a Snack player**

````
```SnackPlayer name=Component/API%20Example

// Example code goes here

```
````

### Submit your PR!

Follow the repo's setup instructions [here](https://github.com/facebook/react-native-website). 

> **Remember** your doc changes will be under facebook.github.io/react-native/docs/**next/**yourcomponent, _not_ facebook.github.io/react-native/docs/yourcomponent

1. Work on master docs (docs/*.md), **not versioned ones** (website/versioned_docs/version-0.*/*.md)
2. Tag your PR with this issue #1579 so it gets properly recognized

### Code Tips

* Extract styles to `StyleSheet` outside of component and at bottom of sample
* Use arrow functions instead of anonymous functions
* Do not leave unused `import`s 
* Sort `import`s alphabetically
* Remember to use semicolons (especially after `export`s)
* Name your Snack "<Component/API>%20Example"
* If the API or component is platform specific, specify which platforms for the Snack player to show like so: ` ```SnackPlayer name=AndroidOnlyAPI&supportedPlatforms=web,android`
* If the API or component has platform specific parts, use `supportedPlatforms` 

Before committing...

* Run `yarn prettier` to ensure your code is made pretty before committing.
* Also `cd website` and `yarn start` to generate any changes to sidebars

## Docs Drive Leaderboard

**🍾🎉🎈Thank you to all these wonderful folks!🍾🎉🎈**

These are the people and their contributions to the docs!

* @ronnyiscoding = { ActivityIndicator : #1590, ImageStyleProps : #1602 }
* @dsznajder = { Button : #1592, TouchableHighlight : #1591 }
* @lfoliveir4 = { FlatList : null}
* @amandeepmittal = { Image : #1582, View : #1582, Share : #1674 }
* @nnajiabraham = { KeyboardAvoidingView : null, VirtualizedList : [#1742, #1632], AppRegistry : #1740 }
* @Darking360 = { Modal : #1605 }
* @kaioduarte = { Picker : #1595, Switch : #1594, TouchableOpacity : #1596, ShadowProps: #1664, Text Style Props : #1710 }
* @espipj = { StatusBar : #1624, TouchableNativeFeedback : #1646, ToastAndroid : #1614, DrawerLayoutAndroid : #1683, Layout Props : #1750 }
* @DaniAkash = { Alert : #1597, Dimensions : #1671, Animated : #1691, LayoutAnimation : #1721, BackHandler : #1685, Transforms : #1613, PanResponder : #1691 }
* @mitulsavani = { View Style Props : #1612 }
* @mdabrowka = { AccessibilityInfo : #1629 }
* @lcelso = { StyleSheet : #1626 }
* @alvessteve = { ImageBackground : #1598 }
* @tsawan = { Text : #1623, AppState : #1666 }
* @Naturalclar = { TextInput : #1584 } 
* @simek = { ActionSheetIOS : #1680, PixelRatio : #1687, Easing : #1682, Settings : #1700 }
* @mohamedsgap = { AppState : #1695, Vibration : #1707, Systrace : #1749 }
* @kakukakug = { TouchableWithoutFeedback : #1686 }
* @ahmdtalat = { Keyboard : #1619 }
* @gedeagas = { Animated : #1702, Flatlist : [#1705, #1729], PermissionsAndroid	: #1720, SectionList : [#1754, #1717] }
* @danilobrinu = { InteractionManager : #1712, Linking : #1751 }

**Comment by amandeepmittal** [comment]
One question, sorry for my naivety, if I am converting a class example into functional example (just converting the snack from class to function) do I keep both the snacks? Also, do I explicitly mention `<block class="functional syntax" />` in each snack?

**Comment by rachelnabors** [comment]
> (I)f I am converting a class example into functional example (just converting the snack from class to function) do I keep both the snacks? 

Yes!

> Also, do I explicitly mention `<block class="functional syntax" />` in each snack?

It's probably best for you to copy the code block under "Convert existing example code to use function components and Hooks," then replace `// Class Example goes here` with the former, class-based example. Then replace `// Function Example goes here` with a function/Hooks example.

Please correct me if I misunderstood your question!

**Comment by amandeepmittal** [comment]
Thank you so much. I understood the point of the whole code block. I should have read the class name on the top that says `toggler`. 😅

**Comment by rachelnabors** [comment]
@amandeepmittal Did you follow steps 3 and 4, too? ;)

**Comment by DaniAkash** [comment]
@rachelnabors Just created a PR for the DatePickerIOS 🗓. I basically rewrote the existing class component into a functional component with hooks. But It clearly has a lot of props, does the Snack have to cover all the props or basic functionality is enough?

**Comment by rachelnabors** [comment]
You know, DatePickerIOS is actually in the process of being _deprecated!_ It sneaked into the table somehow (tsk tsk!). I just removed it. I'm accepting your PR regardless, but go pour your energy into another component who will be sticking around!

**Comment by amandeepmittal** [comment]
@rachelnabors do you have copy-paste version of the matrix table? Also should quote the matrix here or in the PR?

**Comment by DaniAkash** [comment]
It did well for a test subject 😜 will pick up something more vital next...

**Comment by rachelnabors** [comment]
> @rachelnabors do you have copy-paste version of the matrix table? Also should quote the matrix here or in the PR?

@amandeepmittal, are you not able to edit the original post? I tried to put checkboxes in there but couldn't make them work inside tables! Thanks for bearing with me!

**Comment by amandeepmittal** [comment]
> > @rachelnabors do you have copy-paste version of the matrix table? Also should quote the matrix here or in the PR?
> 
> @amandeepmittal, are you not able to edit the original post? I tried to put checkboxes in there but couldn't make them work inside tables! Thanks for bearing with me!

Nope. Can't edit your original post here. No worries.

**Comment by rachelnabors** [comment]
Hmm. Gonna have to think of a way for people to claim each task..

**Comment by amandeepmittal** [comment]
> Hmm. Gonna have to think of a way for people to claim each task..

Can we do this in a gist and link it here? Just a suggestion 🤷‍♂️

**Comment by rachelnabors** [comment]
I added checkboxes :) Just tick one for each task you claim. I'll update them to bold checks when the PRs are merged!

**Comment by jeremy-deutsch** [comment]
I’d like to claim the `Animated` API up-to-date; how do I check off one of those boxes? Maybe it just doesn’t work since I’m on mobile

**Comment by DaniAkash** [comment]
> I’d like to claim the `Animated` API up-to-date; how do I check off one of those boxes? Maybe it just doesn’t work since I’m on mobile

I'd like to pick this one up too. Given the size of the API, can we share this between us?

**Comment by rachelnabors** [comment]
Why don't you both split the load? I'll put your names down for the different tasks (since the checkboxes aren't working!). How do you want to split it?

**Comment by mitulsavani** [comment]
Hi there, I would like claim for View Style Prop.

**Comment by jeremy-deutsch** [comment]
@rachelnabors I could document the API for `Animated.Value`/`Animated.ValueXY`, and leave the rest of `Animated` to @DaniAkash

**Comment by rachelnabors** [comment]
@mitulsavani You got it! Don't forget to reference this issue in your PR!

**Comment by codecog** [comment]
@rachelnabors feel free to assign `<SectionList />` to me!

**Comment by alvessteve** [comment]
@rachelnabors I'll take the <ImageBackground /> :)

**Comment by dsznajder** [comment]
@rachelnabors I'll take `TouchableHighlight` :)

**Comment by ronnyiscoding** [comment]
@rachelnabors here goes my first PR, https://github.com/facebook/react-native-website/pull/1590

**Comment by Darking360** [comment]
@rachelnabors Can I take `Modal`? Thank you :heart:

**Comment by ronnyiscoding** [comment]
@rachelnabors I can take `Image Style Props`  ? Thanks 👍

**Comment by kaioduarte** [comment]
@rachelnabors Can I take `Switch`? Thanks!

**Comment by dsznajder** [comment]
@rachelnabors I'll take `Button` next :)

**Comment by amandeepmittal** [comment]
@rachelnabors I'll work on `View` to add snack and functional component example.

**Comment by rachelnabors** [comment]
Thanks for volunteering, everyone! I've updated the matrix with all your commitments. For anyone who hasn't got a component or API to work on, we still have plenty left! Jump on in!

**Comment by kaioduarte** [comment]
@rachelnabors I did also a small adjustment in `Picker` component example. :grin:

**Comment by DaniAkash** [comment]
@jeremy-deutsch cool I'm up for the task ✌️

**Comment by lfoliveir4** [comment]
@rachelnabors 

Hello! I am super interested and making this contribution.
As I understand it, is it necessary to make an example with hooks and classes referring to the API?

Example: Let's go up and get the Pixel Ratio
Do I have to do the example in the snack using Classes and hooks? Right?

If not, please let me know!

I'm waiting!

**Comment by DaniAkash** [comment]
@rachelnabors Just pushed out an example for the Alert component. Clipboard, Dimensions, Keyboard & LayoutAnimations are next! 

Also will be picking up the Animated API which will be quite a challenging task! 😁

**Comment by DaniAkash** [comment]
I'm unable to get the `Clipboard` working on the Expo web player. Could use some help with https://github.com/facebook/react-native-website/pull/1599

**Comment by ahmdtalaat** [comment]
Hello 
FOA navigating the **table** is quite _tedious_, can you please make it a little bit easier.
I would like to work on the `Keyboard`.

**Comment by DaniAkash** [comment]
> Hello
> FOA navigating the **table** is quite _tedious_, can you please make it a little bit easier.
> I would like to work on the `Keyboard`.

@ahmdtalaat I'll be leaving the `Keyboard` in your care 👍 Picking up `LayoutAnimations` next!

**Comment by DaniAkash** [comment]
@rachelnabors it'd be great if you can create a PR column to the Components and APIs update progress table and list the PRs raised related to the component. It'll greatly help the PRs to be peer-reviewed and tracked neatly ✏️

**Comment by ronnyiscoding** [comment]
@rachelnabors Can I take `VirtualizedList` ?

**Comment by brunokiafuka** [comment]
@rachelnabors, I took the clipboard API.

**Comment by DaniAkash** [comment]
> @rachelnabors, I took the clipboard API.

@brunokiafuka There's already a PR open for this API - https://github.com/facebook/react-native-website/pull/1599

**Comment by brunokiafuka** [comment]
> > @rachelnabors, I took the clipboard API.
> 
> @brunokiafuka There's already a PR open for this API - #1599

Sorry @DaniAkash, I've missed that

**Comment by DaniAkash** [comment]
> > > @rachelnabors, I took the clipboard API.
> > 
> > 
> > @brunokiafuka There's already a PR open for this API - #1599
> 
> Sorry @DaniAkash, I've missed that

@brunokiafuka I can use your help in it 👍 it is WIP

**Comment by hugobytes** [comment]
@rachelnabors feel free to assign LayoutAnimation Snack example to me!

**Comment by DaniAkash** [comment]
> @rachelnabors feel free to assign LayoutAnimation Snack example and Hooks example to me!

@hugobytes I'm almost done with the LayoutAnimation. Will be sending a PR soon.

**Comment by nnajiabraham** [comment]
Hi @rachelnabors can I get the KeyboardAvoidingView. I will be working on _Example uses Snack_ and _Example uses function components and Hooks for that componenet_

**Comment by tsawan** [comment]
Hi @rachelnabors, I will be taking the Text and working on Example uses function components and Hooks.

**Comment by nnajiabraham** [comment]
@ronnyiscoding Are you working on the VirtualizedList precisely on `Example uses Snack and `Example uses function components and Hooks` for that component, if not can I take it from you?

**Comment by ronnyiscoding** [comment]
> @ronnyiscoding Are you working on the VirtualizedList precisely on `Example uses Snack` and `Example uses function components and Hooks` for that component, if not can I take it from you?

ok, it is fine.

**Comment by nnajiabraham** [comment]
Ok then @rachelnabors  I'll be taking VirtualizedList from @ronnyiscoding precisely on `Example uses Snack` and `Example uses function components and Hooks`

**Comment by ronnyiscoding** [comment]
> Ok then @rachelnabors I'll be taking VirtualizedList from @ronnyiscoding precisely on `Example uses Snack` and `Example uses function components and Hooks`

@nnajiabraham I mentioned you in this PR https://github.com/facebook/react-native-website/pull/1607

**Comment by DaniAkash** [comment]
@rachelnabors Summary of my PRs
LayoutAnimation - https://github.com/facebook/react-native-website/pull/1609 [WIP]
Clipboard - https://github.com/facebook/react-native-website/pull/1599 [WIP]
DatePickerIOS - https://github.com/facebook/react-native-website/pull/1583
Dimensions - https://github.com/facebook/react-native-website/pull/1601
Alert - https://github.com/facebook/react-native-website/pull/1597

It'd be helpful if you can add these to the Components and APIs update progress. Also, I'll be picking up `Animated` library next @jeremy-deutsch look forward to my PR soon 👍

**Comment by miganga** [comment]
@rachelnabors I would like to take on Layout Props api to update.  Thank you.

**Comment by rachelnabors** [comment]
@lfoliveir4 :

> As I understand it, is it necessary to make an example with hooks and classes referring to the API?

You only need to make examples with function components and Hooks. If there is ALREADY an example with class components, then use the toggler code to show both examples together :) You should only make new examples with function components/hooks

**Comment by rachelnabors** [comment]
Folks, I have updated the matrix to reflect the projects you've claimed! Phew! Time to review some of your pull requests! Thanks so much for contributing, and if you haven't contributed yet, just claim a component here!

**Comment by lcelso** [comment]
I'll take the Linking

**Comment by Simek** [comment]
I was reviewing today a few PRs related to this issue and I would like to summarize some tips according to code quality and best practices of those new examples. 

There are minor things but overall this code could be used by many developers and mistakes or negligence here could be replicated further. There is no bad intent I just assume that we all want to have the best examples here 😉 

Please pay a bit attention for those details if possible:
* work on master docs (`docs/*.md`) not versioned ones (`website/versioned_docs/version-0.*/*.md`)
* extract styles to `StyleSheet` outside of component if necessary
* use arrow function instead of `function` keyword
* sort `imports` alphabetically
* remember about semicolons (especially after `export`s)
* use same quotation marks across whole example (I don't know which are prefered here `"` or `'` but be consistent)
* remember about specifying `supportedPlatforms` for the Snack if component requires that (example: c7c7929dfa0c20af5bf953c45708ce21ca0019f8)
* brake lines even more often than in your IDE - Snack code block fits only `66` characters per line
* do not overcomplicate Snack names

That's all from me. Thank you all for your contributions. Keep up the good work!

**Comment by lcelso** [comment]
@Simek 

I agree with you, however I have some points in which I wanted us to reflect.

We are using SnackPlayer to generate the example together with the Expo.

Strengths:

Code editor
Demo for three versions, web / ios and android

Negative points:

Project structure using the docusaurus, leaves devs a little limited, since these are markdown files, the examples in react within it are without the sixtaxe highlight.

Perhaps there is some configuration in which we can make the prettier to solve this.
In js and html files I already made certain settings for this.
However, we got to the point of using markdown syntax together with React (SnackPlayer).

What I thought, was maybe to rethink the architecture of the project maybe using gatsbyjs, in which we have integrations with github pages, among other points.

Searching here I saw that this subject was addressed in another post as well and that this is not currently in our plans.
But I think the debate is also worth it, right?

**Comment by mdabrowka** [comment]
I will work on refactoring `AccessibilityInfo` example into a function component with Hooks

**Comment by Simek** [comment]
@lcelso I agree that developing examples could be a bit more difficult due to fact that code is embedded in Markdown. Having examples in separate JS files would be nice (for example we can setup linter than etc.) but changing project architecture and converting all docs could be a challenging and daunting process. I don't even know if it's possible within Docusaurus. Also switch to Gatsby is not likely since Docusaurus is a Facebook product and Gatsby is not (also purpose of those platforms is bit different).

Many code editors/IDEs this days can highlight JS code properly inside other type files ([screenshot](https://user-images.githubusercontent.com/719641/74085408-fd095680-4a78-11ea-91a5-5e607acfdb6e.png)). Running website on localhost and developing with live preview also helps a lot.

But back to main topic. Since `Clipboard` API is being deprecated (#1608) I would like to propose removal of this API from the TODO list. I'm sorry for that to the #1599 and #1604 authors.

It's also a good opportunity to start conversation about removing ALL existing examples from the deprecated components (leaving short description and reference only) before next major release. This whole idea started here https://github.com/facebook/react-native-website/pull/1583#issuecomment-583345444. 

Few pages in the `master` docs right now follow this pattern (ImageEditor, PickerIOS, Slider) but most of deprecated pages include examples. In my opinion removing those examples can force (mainy new) users to follow deprecation notice and use the proper maintained and up to date package which always contains an example or a few of them in the README.

**Comment by espipj** [comment]
Hey @rachelnabors ! I'll be making some changes in the TextInput API docs 😄

**Comment by DaniAkash** [comment]
While testing out the Animated API, I had to write some transforms and realized I might as well use them as an example for the Transforms API. However, as per https://github.com/facebook/react-native/issues/27649, the skew transform isn't working on Android.
## Skew on iOS: 
<img width="224" alt="Screenshot 2020-02-09 at 7 13 07 AM" src="https://user-images.githubusercontent.com/6841445/74094922-e6154380-4b0e-11ea-9469-729736570b06.png">
## Skew on Android:
<img width="218" alt="Screenshot 2020-02-09 at 7 12 29 AM" src="https://user-images.githubusercontent.com/6841445/74094930-06450280-4b0f-11ea-9912-ce5dfbd256d2.png">

**Comment by ahmdtalat** [comment]
Hello guys,
I've an issue and need you help. when I make a change to the docs. it does not render although I'm using the latest version as the documentation says `Changes to /docs will only be visible in the latest version of the documentation (master).
`.
to solve this. I have to run `yarn run version 0.62`. but is that's ok?
I mean what if I want to push changes. should I push the docs changes only?

**Comment by DaniAkash** [comment]
@ahmdtalat the changes are in the master documentation. 

To open master documentation: 
you can directly click on the version number next to the "React Native" title icon of the page and see the list of all versions including the master documentation. You can view your changes in this section. No need to run version 0.62 command
<img width="269" alt="Screenshot 2020-02-10 at 12 20 45 PM" src="https://user-images.githubusercontent.com/6841445/74127251-1be62500-4c00-11ea-85c3-0dd6268f21dc.png">

**Comment by rachelnabors** [comment]
I've updated the matrix with new contributors and some new protips to help folks contributing their code (thanks, @Simek!) Please check them out under "Submit your PR!" and "Code Tips"! Thanks again to everyone for contributing!

**Comment by lfoliveir4** [comment]
> @lfoliveir4 :
> 
> > As I understand it, is it necessary to make an example with hooks and classes referring to the API?
> 
> You only need to make examples with function components and Hooks. If there is ALREADY an example with class components, then use the toggler code to show both examples together :) You should only make new examples with function components/hooks

How do I stop checking the lib that I'm going to do the example?

**Comment by lfoliveir4** [comment]
> > @lfoliveir4 :
> > > As I understand it, is it necessary to make an example with hooks and classes referring to the API?
> > 
> > 
> > You only need to make examples with function components and Hooks. If there is ALREADY an example with class components, then use the toggler code to show both examples together :) You should only make new examples with function components/hooks
> 
> How do I stop checking the lib that I'm going to do the example?

@rachelnabors 
I will make the contribution with the Image and FlatList component

**Comment by DaniAkash** [comment]
> I've updated the matrix with new contributors and some new protips to help folks contributing their code (thanks, @Simek!) Please check them out under "Submit your PR!" and "Code Tips"! Thanks again to everyone for contributing!

@rachelnabors add me to the Transforms API, I have already raised a PR https://github.com/facebook/react-native-website/pull/1613

**Comment by ahmdtalat** [comment]
Hello Guys, I added Keyboard API example using function components & hooks #1619

**Comment by mohamedsgap** [comment]
Can't find TabBarIOS in [react-native/Libraries](https://github.com/facebook/react-native/tree/9caf99162dcd1ee73c571c494ca3115540f45f69/Libraries/Components) to start update its API

**Comment by lcelso** [comment]
@Simek e @rachelnabors 

I understand your point and agree that we should have a scenario where editing the javascript files is a little easier.

I was analyzing some PRs here and I miss some standards, like 2 tabs, single quotes among other points, in which I believe we can define this here as a team, what do you say?

On the points that you raised from discontinuation I really totally agree.

**Comment by lcelso** [comment]
Hi guys, can someone review this for me?

@Simek @rachelnabors 
# 1626

**Comment by tsawan** [comment]
@rachelnabors I can also change "Image" and "AppState" examples to use functional components and hooks.

**Comment by luism3861** [comment]
@rachelnabors  this is my 2 pr's #1627  #1617

**Comment by luism3861** [comment]
@rachelnabors  #1617  #1627  !!

**Comment by DaniAkash** [comment]
I'm struggling to get the RNTester app running in my machine. Anyone else facing issue with this app?

**Comment by rachelnabors** [comment]
Hi everyone, sorry for the absence. Got sick again!

Matrix is updated!

**Comment by rachelnabors** [comment]
@lcelso We have a discord we could use to coordinate, but then we'd have to track in two locations, which historically can be even more confusing! If you want to team up with someone, I suggest you reach out to them personally and choose the format of your choice to coordinate :) Could you expand on "do you think we can have some language context? Mainly in the context for examples."?

**Comment by mitulsavani** [comment]
Hi @rachelnabors, I see that you recently updated the metrics and tick marked `View Style Props`  referenced #1614 on it. I am not sure if that was a typo if so then, my PR is #1612 FYI. 

I wonder if there is anything else that I need to finish for `View Style Props`? If not then, I can start working on other pending work

**Comment by lcelso** [comment]
@rachelnabors Hello, I believe that a faster communication channel could speed up some points, such as clearing doubts, which we can remove / discontinue.

Especially between some points of decisions and this done we can make official here.

About languages, I was seeing react.org and this is already done there.

This is a point if we have a group on Slack or anywhere else that we will have a more fluid communication.

We can discuss tools or other things like that and I try to have a better agility to give things a better flow.

What do you say?

**Comment by amandeepmittal** [comment]
Hi @rachelnabors, I will update the `Share` API by adding functional and hooks example since it already has the class component example.

**Comment by Simek** [comment]
@rachelnabors Could you asses the idea proposed in https://github.com/facebook/react-native-website/issues/1579#issuecomment-583735028?

**Comment by DaniAkash** [comment]
> @lcelso I agree that developing examples could be a bit more difficult due to fact that code is embedded in Markdown. Having examples in separate JS files would be nice (for example we can setup linter than etc.) but changing project architecture and converting all docs could be a challenging and daunting process. I don't even know if it's possible within Docusaurus. Also switch to Gatsby is not likely since Docusaurus is a Facebook product and Gatsby is not (also purpose of those platforms is bit different). Many code editors/IDEs this days can highlight JS code properly inside other type files ([screenshot](https://user-images.githubusercontent.com/719641/74085408-fd095680-4a78-11ea-91a5-5e607acfdb6e.png)). Running website on localhost and developing with live preview also helps a lot.
> 
> **But back to main topic.** @rachelnabors Since `Clipboard` API is being deprecated (#1608) I would like to propose removal of this API from the TODO list (I would like to apologize authors of #1599 and #1604 for that).
> 
> It's also a good opportunity to start conversation about removing ALL existing examples from the deprecated components (leaving short description and reference only) before next major release. This whole idea started here [#1583 (comment)](https://github.com/facebook/react-native-website/pull/1583#issuecomment-583345444) but I think that this thread is a better place for this conversation.
> 
> Few pages in the `master` docs right now follow this pattern (ImageEditor, PickerIOS, Slider) but most of deprecated pages include examples. In my opinion removing those examples can force (mainy new) users to follow deprecation notice and use the proper maintained and up to date package which always contains an example or a few of them in the README. There are more benefits to users regarding to compatibility, or better support with issues for those packages but I would like also to hear you opinion about this idea.

While it is a good idea to remove the options from the sidebar, I still feel removing the examples would be bit harsh for people who are already heavily using these components...

**Comment by rachelnabors** [comment]
@simek I can confirm that deprecated components will be removed from the sidebar but their legacy pages will live on quietly—there is no harm in finishing up the examples! I hope this settles things.

@amandeepmittal you're on Share, thanks!

@lcelso Will need to hear more support from other people for your idea—worry another communication channel now would slow down and split communications rather than keeping us moving in one direction with one source of truth—this location. Feel free to coordinate privately on individual collaborations, naturally.

@mitulsavani I'll correct the issue post haste.

Next up: I'll be removing pieces of the matrix that have been completed to make it easier to see what's left to do. Thanks, everyone, for your hard work so far!

**Comment by Simek** [comment]
@rachelnabors Are there any planned changes for the [Components and APIs](https://facebook.github.io/react-native/docs/next/components-and-apis) page? Would you like to mark in some way or even remove deprecated pages listed in there?

**Comment by rachelnabors** [comment]
@Simek Smart thinking! And to think I've been looking at this page the whole week without realizing it! What a cold! Good thing you're on the ball!

We should remove the deprecated components and APIs from the page as we are removing them from the sidebar as well. Would you like to handle that one?

**Comment by lcelso** [comment]
@rachelnabors @Simek 

I created a channel for quick communication and central points of doubts here. 

[link to slack](https://join.slack.com/t/react-native-website/shared_invite/enQtOTU5NzkzMjA1MzE3LTEzNjBhYTdlNGVlZGEzNTEzZTlkNTliMTBjOTZlN2RiOWQ3MjMyMzjMjMj)

However I agree that here we must have the data and everything that was talked about there registered here.

What do you say?

Because I think so, I am feeling our agility / decision making is very slow.

There are points of doubt about how we should do it and even points of discussion with better fluidity.

We currently have many PRs stopped a lot of them for many days. Among other points where we can try to start reviewing this.

**Comment by Simek** [comment]
@rachelnabors I think that "API" link in top bar which leads to `ActivityIndicator` instead of  `Components and APIs` in `next` docs do not help either 😉 

Back to the deprecation discussion - I would like to help with those changes but I'm still not 100% sure how to perform them. 

In my opinion, in some point in time `0.61` docs should be regenerated to include all the work from this issue and more changes that landed on the `next` docs during last months (I assume this should happen before or on `0.62` release?). 

Merging sidebar and `Components and APIs` deprecation cleanup PRs before refreshing `0.61` will cause a loss of some improvements (mainly deprecation indicators and notes because they are currently only on the `next` docs and are targeted to be deleted before/on `0.62` release). That is also the reason why I was talking about generating `0.62` docs when you asked about sidebar changes in earlier discussion.

**Comment by Simek** [comment]
@lcelso I think that Slack/Teams/Discord channel will help with communication but you should remember that this is still a Facebook repo which is not officially maintained by the community. 

In my opinion creating this kind of communication platform should be approved and managed by FB employees (maintainers). If your idea got a green light of course I will join along.

**Comment by mitulsavani** [comment]
> @lcelso I think that Slack/Teams/Discord channel will help with communication and can streamline PR processes but you should remember that this is still a Facebook repo which is not officially maintained by the community.
> 
> In my opinion creating this kind of communication platform should be approved and managed by FB employees (maintainers). If your idea got a green light of course I will join along.

I definitely agree with @Simek opinion,  @lcelso I recommend you to open an issue on core react native repo to discuss further about it.

**Comment by lcelso** [comment]
Hello guys,

I agree for this I opened the discussion here with X and she gave me this answer.

> @lcelso Will need to hear more support from other people for your idea—worry another communication channel now would slow down and split communications rather than keeping us moving in one direction with one source of truth—this location. Feel free to coordinate privately on individual collaborations, naturally.

And if this suggestion is a bad point I would like to understand how we can resolve the points below:

- Long standing PRs
- Doubts of those who want to help and have a quick flow of responses
- Discussions on topics that we can make obsolete or not

**Comment by lcelso** [comment]
> @rachelnabors Acho que o link "API" na barra superior que leva à `ActivityIndicator`página em vez da `Components and APIs`página nos `next`documentos também não ajuda😉
> 
> De volta à discussão sobre descontinuação - eu gostaria de ajudar com essas alterações, mas ainda não tenho 100% de certeza de como executá-las.
> 
> Na minha opinião, em algum momento os `0.61`documentos devem ser regenerados para incluir todo o trabalho desse problema e mais alterações que chegaram aos `next`documentos nos últimos meses (presumo que isso deva acontecer antes ou no `0.62`lançamento?).
> 
> A mesclagem de `Components and APIs`PRs de limpeza da barra lateral e de descontinuação antes da atualização `0.61`causará uma perda de algumas melhorias (principalmente indicadores e notas de descontinuação, porque eles estão atualmente apenas nos `next`documentos e devem ser excluídos antes / no `0.62`lançamento). Essa também é a razão pela qual eu estava falando sobre a geração de `0.62`documentos quando você perguntou sobre alterações na barra lateral em discussões anteriores.

I would like to help with this too

**Comment by cosio55** [comment]
Hi @rachelnabors I was looking at the missing components and I saw that TabBarIOS & TabBarIOS.item were deprecated on RN 0.5959 here's the [CHANGELOG](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#059) they don't appear on the docs for version 0.59 but they re appeared on version 60 and 61.

**Comment by rachelnabors** [comment]
@cosio55 Thank you for hunting that one down! I wish we could go back and remove it from all the following versions of the docs now. I'll remove it from the matrix. Much obliged!

**Comment by rachelnabors** [comment]
I've updated the matrix and moved all the completed components and APIs to a leaderboard to recognize their contributors. Thank you everyone! There's still lots to do: those APIs need love, too!

**Comment by nnajiabraham** [comment]
Hi @rachelnabors, please can you assign me for all fields on AppRegistry, will update API and write examples 🙌🙌

**Comment by kaioduarte** [comment]
@rachelnabors I'll take `Shadow Props` and `Text Style Props` :wink:

**Comment by ahmdtalat** [comment]
@rachelnabors update my username in the matrix from (Keyboard) ahmdtalaat -> @ahmdtalat 
and I need someone to review my PR #1619, thanks.

**Comment by espipj** [comment]
Appreciate the gifs lol 🦄

**Comment by espipj** [comment]
Appreciate the gifs lol 🦄 @rachelnabors

**Comment by lfoliveir4** [comment]
@rachelnabors What do I need to do to finish my contribution with FlatList and if it is possible to update another API or Component?

**Comment by rachelnabors** [comment]
Matrix is updated! Thanks everyone!

@lfoliveir4 just pick another one and get to it! Flatlist is in the queue. There area lot of PRs to merge. Thank you for your patience!

**Comment by Simek** [comment]
@rachelnabors Can you claryfiy how you would like to perform  changes to deprecated components mentioned in https://github.com/facebook/react-native-website/issues/1579#issuecomment-588361038? I have created an explanation why I'm not sure how to do this properly here: https://github.com/facebook/react-native-website/issues/1579#issuecomment-588812978. If my explanation is still foggy or confusing instead of clarifying I can try explain it again in different words.

**Comment by rachelnabors** [comment]
@simek Changes to versioned docs have to happen in */website/versioned_docs/<version number>/*. Add the <file>.md with the content you want to edit for that version. More information on how to do that is here: https://docusaurus.io/docs/en/next/versioning 

Are you on Discord or Twitter? Perhaps there or in a new issue would be the best place to coordinate, as this channel is for people updating the docs and we could confuse them with conversation about versioning files :)

**Comment by Simek** [comment]
@rachelnabors Sorry for bringing up this topic here. I have some basic knowledge about versioning in Docasurus but in my opinion the right process depends on final effect you want to archive. We can continue discussion on the Discord, since I'm not a frequent user of my Twitter. My Discord handle is `Simek#2069`.

**Comment by kakukakug** [comment]
Hi @rachelnabors, I Want to contribute React Native!
please can you assign me for TouchableWithoutFeedback: Modify the example to use snacks.

**Comment by tsawan** [comment]
@rachelnabors I've just checked and the following component examples are already using functional components and hooks so no further action is required.

- InputAccessoryView
- RefreshControl
- SafeAreaView
- ScrollView

**Comment by tsawan** [comment]
@rachelnabors I've also checked that almost all of the remaining APIs don't need any work for "Examples uses functional components and hooks". So these can be marked as Done or NA.

**Comment by rachelnabors** [comment]
@kakukakug You're on it!

@tsawan Thanks for checking on all those! I've removed those four components from the matrix. I'm happy to tick off the APIs—but could you be more specific than "almost all?" Who's missing yet?

**Comment by ahmdtalat** [comment]
Hello guys, I'm working on the Keyboard API, I added hooks example and snacks, but how to check if the `API is up to date`? #1619

**Comment by tsawan** [comment]
@rachelnabors following APIs don't need anything for "Examples uses functional components and hooks":  ActionSheetIOS, Easing, InteractionManager, Settings, Systrace.
I am not too sure about BackHandler API, so you may want to check this.

**Comment by rachelnabors** [comment]
@tsawan Hmm, they may have function components in their example code, but they still need big example Snacks that use them as well. So they stay up!

**Comment by lfoliveir4** [comment]
> Matrix is updated! Thanks everyone!
> 
> @lfoliveir4 just pick another one and get to it! Flatlist is in the queue. There area lot of PRs to merge. Thank you for your patience!

@rachelnabors So I just have to wait for PR? Can I contribute with another API if I have it available?

**Comment by codecog** [comment]
Will get to SectionList this weekend

**Comment by rachelnabors** [comment]
> So I just have to wait for PR? Can I contribute with another API if I have it available?

@lfoliveir4 You sure can :) Don't wait on my PRs. Sometimes there's feedback, but it's rare that that should stop you from tackling a new task!

@codecog Oh thank you so much!

**Comment by rachelnabors** [comment]
*Folks!* We're hoping to cut the next branch of the docs in a few weeks. Your contributions are making this a great new edition of the React Native docs! Thank you so much for your hard individual efforts, from the tiniest example update to whole new pages! 

If you have the bandwidth, taking on those APIs are going to help push us over the edge to completeness. If you've updated a component already, I know you have what it takes to tackle and API. Hit me up here, on Twitter, or Discord if you run into any trouble! 

Let's do this!
![shera_BAM](https://user-images.githubusercontent.com/236306/75556115-7de2bf00-5a35-11ea-9ac1-f6dbc6f3ce20.gif)

**Comment by rachelnabors** [comment]
Oh snap! We are so close to finishing _all the Core Components!_

* @kakukakug, we're counting on you for `TouchableWithoutFeedback`
* @ronnyiscoding, can you check `VirtualizedList` to see if it's API is up to date?
* _We need a volunteer for `DrawerLayoutAndroid`!_ Any takers?

**Comment by espipj** [comment]
If not worked/assigned yet I could take DrawerLayoutAndroid @rachelnabors

**Comment by ahmdtalat** [comment]
@rachelnabors Hello Rachel, I've updated the keyboard API and now I think it's done, check the changes. :) #1619

**Comment by rachelnabors** [comment]
@espipj It's all yours! 

Matrix is updated!

**Comment by lcelso** [comment]
@rachelnabors and @Simek 

How do i find you on discord?

**Comment by mohamedsgap** [comment]
Hey @rachelnabors I gonna check if the 'AppState' API is up to date or not! and I gonna update it if it needs.

**Comment by rachelnabors** [comment]
@lcelso I'm 'R "Nearest" Nabors#7453' on Discord

@mohamedsgap THANK YOU! Please report back here if it needed updating or not so I can update the matrix!

**Comment by DaniAkash** [comment]
@rachelnabors Just added examples for Animated & PanResponder. If there are any specific animations you think I need to cover, let me know 😁

**Comment by mohamedsgap** [comment]
Hey @rachelnabors I've just checked the core code of 'AppState' and I think it's up to date but there is an event in core code called 'memoryWarning' didn't mention in the events' reference in the doc, do you need me to fill a PR for it? I'm ready ;)

**Comment by rachelnabors** [comment]
Yes please, @mohamedsgap! That would be an API in need of updating! Please PR <3

**Comment by rachelnabors** [comment]
@espipj, @codecog, @ronnyiscoding: How are your components doing? We are waiting on your three components to close out the updated components docs! We're so close! Don't stop now!

Everyone else: the matrix is up to date. We've still got some APIs in need of examples and updates! If you have the time to commit to working on one, it really helps the docs!

**Comment by danilobrinu** [comment]
@rachelnabors how I can help ?

**Comment by mohamedsgap** [comment]
Hi @rachelnabors, very excited to contribute more, so I gonna add an example uses Snack for 'Vibration' API, stay tuned for the PR ;)

**Comment by Simek** [comment]
@mohamedsgap `Vibration` API has been updated in 5fcd97d80e58cfadc81ff0bdc67b3c53fe177e61.

**Comment by rachelnabors** [comment]
@mohamedsgap Actually, the example is still in class components. If you could add a function component exampe and a class/function component toggler I can remove it from the matrix! Won't you please? I'd have less to worry about!

**Comment by rachelnabors** [comment]
@danilobrinu Good to hear your enthusiasm! Read that post up top and claim an API to update down here!

**Comment by danilobrinu** [comment]
@rachelnabors I want to take the Settings :)

**Comment by Simek** [comment]
@danilobrinu I have PR in stash for the `Settings` but there is an issue with API that need to be resolved first: facebook/react-native#28213

**Comment by danilobrinu** [comment]
@Simek ohh Ok

**Comment by rachelnabors** [comment]
I updated the matrix to show that!

**Comment by mohamedsgap** [comment]
Okay @rachelnabors I will update the entire 'Vibration' API, then I gonna create PR for it asap :)

**Comment by rachelnabors** [comment]
We still have three APIs in need of owners!

* [PermissionsAndroid](https://reactnative.dev/docs/permissionsandroid) and [InteractionManager](https://reactnative.dev/docs/interactionmanager) need their examples wrapped in a Snack player

* [Systrace](https://reactnative.dev/docs/systrace) needs an example

**Comment by rachelnabors** [comment]
@espipj, @codecog, @ronnyiscoding,	@jeremy-deutsch, @DaniAkash, @nnajiabraham, @ahmdtalat, @miganga, @lcelso, @Browndini, @Simek, @kaioduarte, @mohamedsgap: 

Thank you all for contributing your hours to this effort so far! I've merged all but a few PRs, and your components and APIs are the last ones we need to complete this effort! 

If you have over-committed (who hasn't?) and cannot find the time to work on your adopted component/API before next Friday, please post below so I can release the component/API for someone else to take on <3 

> Any components without PRs by next Friday will be released for someone else to take on.

You've done a great job so far! The docs are rocking for all your help! I could not be prouder!

**Comment by danilobrinu** [comment]
@rachelnabors can you give one of those :D ? I'm open to help

**Comment by rachelnabors** [comment]
Sure, @danilobrinu! I give you `InteractionManager`!

**Comment by danilobrinu** [comment]
@rachelnabors thanks :)

**Comment by ahmdtalat** [comment]
@rachelnabors  Does the Keyboard component need extra work? my PR is merged.

**Comment by rachelnabors** [comment]
@ahmdtalat You did contribute an example! But did you also check to see if the API needed updating?

**Comment by danilobrinu** [comment]
@rachelnabors I run the `yarn start` command but I don't see my changes in the page. I see the page is refreshed everytime I do some change but I don't see my changes.  How should works ?

**Comment by ahmdtalat** [comment]
my last commit was about updating the API. https://github.com/facebook/react-native-website/pull/1619/commits/27ec6e8df4d0dc0e1969d90f61d9e4ff6e37542b

**Comment by mohamedsgap** [comment]
@danilobrinu Hi Danilo, to see your changes on Docs you should, put 'next/yourComponentName' in the URL, to learn more please, take a look at contribution guides.

@ahmdtalat Hey Ahmed, Make sure that you check the core code of 'Keyboard' API component, if there any change in the core code, you need to mention it in doc as an API reference.

**Comment by danilobrinu** [comment]
@mohamedsgap thanks

**Comment by ahmdtalat** [comment]
@mohamedsgap I did if I'm not mistaken. this the new function I added to the docs:
```jsx
static scheduleLayoutAnimation(event)
```

Useful for syncing TextInput (or other keyboard accessory view) size of position changes with keyboard movements.

**Comment by mohamedsgap** [comment]
@ahmdtalat If you already mentioned it in the doc as an API reference, that’s good and tell Rachel to update the matrix, if not please update it, and if u need any help, just ping me ✌️🙂

**Comment by rachelnabors** [comment]
@danilobrinu are you looking at http://localhost:3000/react-native/docs/next/YOURPAGE ?

@ahmdtalat Then you're good to go! I'll remove it from the matrix! Thank you!

**Comment by danilobrinu** [comment]
@rachelnabors yes, is my bad (typo mistake) hehehe sorry.

**Comment by gedeagas** [comment]
Can i take `PermissionsAndroid` @rachelnabors

**Comment by rachelnabors** [comment]
@gedeagas Sure thing! It's a little weird to use with Expo (see also the bit on the page about using it with Expo!). Here's a sample example from @brentvatne: https://snack.expo.io/rJZTlmgrI

**Comment by rachelnabors** [comment]
@gedeagas re: #1705, are there anymore updates that the API needs? Can I remove it from the list of work that needs doing?

**Comment by danilobrinu** [comment]
@rachelnabors can you review my PR #1712, I think is done but I prefer your feedback :)

**Comment by rachelnabors** [comment]
@danilobrinu just a few more edits! Please check your PR for more info <3

**Comment by gedeagas** [comment]
@rachelnabors thank you for merging the pr, there is no more update that the Flatlist API needs.
 
And also about the `PermissionsAndroid`, as you say because snack use expo under the hood It's a little weird to use and I notice that while the popup for `requestCameraPermission` does come up the `title`, `message`, and `button` don't. Is that okay?

**Comment by codecog** [comment]
@gedeagas @rachelnabors please take section list from me if you would like another one.

**Comment by gedeagas** [comment]
Thanks @codecog, i think i will take over `section list` first until i have clarification about `PermissionsAndroid`.

**Comment by rachelnabors** [comment]
@gedeagas It's ok. It is better than no example :) Moving you on to `SectionList` as well

**Comment by gedeagas** [comment]
Got it thanks @rachelnabors.

**Comment by gedeagas** [comment]
@rachelnabors #1717 and #1720 should be ready for review ❤️

**Comment by mohamedsgap** [comment]
So proud to see the huge progress that all the contributors are made so far, all the components and APIs are already assigned except `systrace` API, I tried to look for the core code of it to figure out it built under the hood but I couldn't find it in the 'react-native' repo, so please if anyone has an idea how to start with `systrace`. I'll be so greatful. @rachelnabors

**Comment by Naturalclar** [comment]
@mohamedsgap 
File for `systrace` api in the react-native repo seems to exist here 
https://github.com/facebook/react-native/blob/master/Libraries/Performance/Systrace.js
I haven't read the code much yet for me to help out, but should be a good starting point :+1:

**Comment by rachelnabors** [comment]
Wow, we're down to 7 docs left to write! If the owners haven't spoken for them by the end of tomorrow, I'll be releasing them back into the pool for more active members to pick up. Things happen! We'll carry the torch!

@mohamedsgap Shall I sign you up for `systrace`? The last kitten in the box, I'm sure it would be happy to find a home with you!

@gedeagas Does `sectionList` still need its API updated? Let me know! Hope we can take it off the list <3

**Comment by nnajiabraham** [comment]
@rachelnabors still working on AppRegistry, just got sidetracked with other stuffs

**Comment by DaniAkash** [comment]
@rachelnabors Transforms PR is merged. The API can be marked closed...

**Comment by mohamedsgap** [comment]
Okay @rachelnabors  I gonna work on `systrace`, I hope ship it asap ;)

**Comment by rachelnabors** [comment]
Folks: in less than 12 hours I will be releasing any docs in the matrix whose people haven't checked in:

SectionList | @gedeagas
VirtualizedList | @ronnyiscoding 
Layout Props | @miganga 
Linking | @lcelso
PanResponder | @DaniAkash 

There's a lot going on and we're really close. If you're still in or if you need to bow out, please just let me know!

**Comment by gedeagas** [comment]
@rachelnabors still searching the `SectionList` codebase for any changes. will update here ASAP.

**Comment by DaniAkash** [comment]
> Folks: in less than 12 hours I will be releasing any docs in the matrix whose people haven't checked in:
> 
> SectionList | @gedeagas
> VirtualizedList | @ronnyiscoding
> Layout Props | @miganga
> Linking | @lcelso
> PanResponder | @DaniAkash
> 
> There's a lot going on and we're really close. If you're still in or if you need to bow out, please just let me know!

@rachelnabors  I'm working on the PanResponder API!

**Comment by rachelnabors** [comment]
The following Components and APIs need new people to work on them! Please give them a good home!

* VirtualizedList needs someone to check its API for updates
* Layout Props and Linking both need someone to add examples and check their APIs for updates

Just comment that you'd like to take them on!

**Comment by danilobrinu** [comment]
@rachelnabors can I take the `Linking` docs task ?

**Comment by nnajiabraham** [comment]
@rachelnabors Working on AppRegistry and just created a PR for the docs update. For the examples, it won't be possible to create an example using SnackPlayer since Snack uses Expo and Expo abstracts that from the User. Is there any alternative to use for creating examples or should I forget about examples in that case???

Also, can I have the VirtualizedList API update, will work on it now?

**Comment by espipj** [comment]
Hey! I'll be taking Layout Props ⏹️ @rachelnabors

**Comment by rachelnabors** [comment]
@nnajiabraham Add some example code snippets using our usual code block notation then!

@danilobrinu You're on Linking now!

@espipj You're on Layout Props

Thank you three very much for your trouble and toil!

**Comment by nnajiabraham** [comment]
Hi, @rachelnabors  all done with VirtualizedList API and AppRegistry example, PRs are #1742 and #1743 respectively. Let me know if anything needs changing 👍

**Comment by danilobrinu** [comment]
Hi @rachelnabors I have a question. I get an error when I call to `Linking.sendIntent` -> `s.default.sendIntent is not a function`. I see the source and is part of the current API so I don't understand why I get that error :'(, can you help me with that ??

**Comment by danilobrinu** [comment]
@rachelnabors can you review this PR #1751 ?

**Comment by rachelnabors** [comment]
@danilobrinu This question is better posited on your PR! Would you mind moving it there? (Also, double check you're using the most recent version of React Native!)

**Comment by rachelnabors** [comment]
I've updated the matrix! Just four left to go!

* `AppRegistry` and `Linking` are already in PR

**Comment by danilobrinu** [comment]
@rachelnabors I tested that code in Snack and I get that error.

**Comment by gedeagas** [comment]
hi @rachelnabors, after searching the codebase I only found one deprecated prop. if anyone finds any changes please let me know and I will happily update the docs

**Comment by rachelnabors** [comment]
Folks, it looks like [`Text.onlayout`](https://reactnative.dev/docs/text#ontextlayout) is still "TODO." Anyone want to patch it up?

**Comment by nnajiabraham** [comment]
@rachelnabors I could patch it up, what is needed for that??

**Comment by rachelnabors** [comment]
Literally replace "TODO" with a proper description of what `onlayout` is and does!

**Comment by DaniAkash** [comment]
@rachelnabors Signing off PanResponder API as up-to-date in https://github.com/facebook/react-native-website/pull/1691

Which means we only have two more to go!

Also the PR for Transforms is at - https://github.com/facebook/react-native-website/pull/1613

**Comment by rachelnabors** [comment]
We are one PR away from being done! Dear contributors, I must ask, what is your favorite format for swag? For me, I love stickers. I put them on my laptop. But some prefer buttons you can wear with pride. What about you?

**Comment by DaniAkash** [comment]
> We are one PR away from being done! Dear contributors, I must ask, what is your favorite format for swag? For me, I love stickers. I put them on my laptop. But some prefer buttons you can wear with pride. What about you?

Gosh, I love buttons! ✨

**Comment by gedeagas** [comment]
@rachelnabors I prefer stickers ❤️, because of recent cancelation of f8 2020 I can't get sticker packs to share with my local Facebook developer circles anymore 😥. But any swag is welcome 😄.

**Comment by mohamedsgap** [comment]
@rachelnabors, I love stickers too :heart:.They enables me to show how much I admire what I put on my laptop :hugs:

**Comment by Naturalclar** [comment]
I love stickers ❤️

**Comment by nnajiabraham** [comment]
@rachelnabors I've got the PR up for the Text.onTextLayout #1757  let me know if any other thing needed 🙌 
Also, discovered my info on the Docs Drive Leaderboard is missing and having some nulls 🤣 to help, here are the PRs I had if you'd want to update the LeaderBoard, Thank You 🙌 ... 
#1606 #1743 #1742 #1740 #1632 #1757 

Finally will love stickers or t-shirts ❤️ and if there are any other ways I could help contribute to this project or any other FB projects, will appreciate your guidance and opportunity on where to start

**Comment by rachelnabors** [comment]
🍾🎉🎈Thank you everyone! We are mission complete!🍾🎉🎈

Thank you all so much for your conributions! The new docs are being cut now, meaning all your contributions will be front and center, not hiding under **/next/**!

If you could be so kind to [give me a little info about yourselves in this form,](https://docs.google.com/forms/d/1h4evpvafv5WVfupr4zohoizZ1nVtk__8U8LH1odQFSI/) I can see about sending you any swag we might be able to acquire (might be awhile, given the current pandemic!)

![mlp_party](https://user-images.githubusercontent.com/236306/76889154-00181380-687d-11ea-9073-f05165ccc0df.gif)

**Comment by danilobrinu** [comment]
@rachelnabors I love t-shirts and stickers ♥️.

**Comment by espipj** [comment]
Amazing effort by everyone! Thanks @rachelnabors for your patience and support with/to everyone! 🚀🦄

**Comment by rachelnabors** [comment]
So, umm, the pandemic has made procuring and mailing swag more difficult than I had imagined! Eek! But we have not forgotten your contributions :) You can expect a shout out in an upcoming blog post!

**Comment by mitulsavani** [comment]
> So, umm, the pandemic has made procuring and mailing swag more difficult than I had imagined! Eek! But we have not forgotten your contributions :) You can expect a shout out in an upcoming blog post!

Hi @rachelnabors, could you please post the link of the blog post here for the reference when the team publishes it?

**Comment by Naturalclar** [comment]
@mitulsavani I believe this is the blogpost :) https://reactnative.dev/blog/2020/07/23/docs-update

**Comment by luism3861** [comment]
> 🍾🎉🎈Thank you everyone! We are mission complete!🍾🎉🎈
> 
> Thank you all so much for your conributions! The new docs are being cut now, meaning all your contributions will be front and center, not hiding under **/next/**!
> 
> If you could be so kind to [give me a little info about yourselves in this form,](https://docs.google.com/forms/d/1h4evpvafv5WVfupr4zohoizZ1nVtk__8U8LH1odQFSI/) I can see about sending you any swag we might be able to acquire (might be awhile, given the current pandemic!)
> 
> ![mlp_party](https://user-images.githubusercontent.com/236306/76889154-00181380-687d-11ea-9073-f05165ccc0df.gif)

hi @rachelnabors  what happened if i have contributed in another issues i have to complete this form form for stickers or t shirt ?
